bazel/repository_locations.bzl

```diff
@@ -75,8 +63,12 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "protobuf-3.5.0",
         urls = ["https://github.com/google/protobuf/archive/v3.5.0.tar.gz"],
     ),
+    com_lightstep_tracer_cpp = dict(
+        commit = "6a198acd328f976984699f7272bbec7c8b220f65",
+        remote = "https://github.com/lightstep/lightstep-tracer-cpp",  # v0.6.1
+    ),
     envoy_api = dict(
-        commit = "2dcc435e8ae1d35f8c3a4fa9f132778482fb1a78",
+        commit = "c15104fce713856203fa4ad948d752e9a1f657c1",
```

Comment:
revert this file other than this line? those seems unrelated.

Reply:
the vscode bazel plugin seems a bit overzealous in its linting...